simulations/vip-502/unichainmainnet.ts

```
@@ -0,0 +1,114 @@
+    });
+
+    describe("Risk parameters", () => {
+      describe(`risk parameters`, () => {
+        let vToken: Contract;
+        let comptroller: Contract;
+
+        before(async () => {
+          vToken = await ethers.getContractAt(VTOKEN_ABI, VUNI_CORE);
+          comptroller = await ethers.getContractAt(COMPTROLLER_ABI, COMPTROLLER_CORE);
+        });
+
+        it(`should set reserve factor to 0.25`, async () => {
+          expect(await vToken.reserveFactorMantissa()).to.equal(parseUnits("0.25", 18));
+        });
+
+        it(`should set collateral factor to 0.5`, async () => {
+          const market = await comptroller.markets(VUNI_CORE);
+          expect(market.collateralFactorMantissa).to.equal(COLLATERAL_FACTOR);
+        });
+
+        it(`should set liquidation threshold to 0.55`, async () => {
+          const market = await comptroller.markets(VUNI_CORE);
+          expect(market.liquidationThresholdMantissa).to.equal(LIQUIDATION_THRESHOLD);
+        });
+
+        it(`should set supply cap to 4000000`, async () => {
+          expect(await comptroller.supplyCaps(VUNI_CORE)).to.equal(parseUnits("4000000", 18));
+        });
+
+        it(`should set borrow cap to 2000000`, async () => {
+          expect(await comptroller.borrowCaps(VUNI_CORE)).to.equal(parseUnits("2000000", 18));
+        });
+
+        it("enter market not paused", async () => {
+          const borrowPaused = await comptroller.actionPaused(VUNI_CORE, Actions.BORROW);
+          expect(borrowPaused).to.equal(false);
+        });
+      });
+    });
+
+    it("Interest rates", async () => {
+      checkInterestRate(
+        interestRateModelAddresses,
+        "VUNI",
+        {
+          base: "0",
+          multiplier: "0.15",
+          jump: "3",
+          kink: "0.45",
+        },
+        BLOCKS_PER_YEAR,
+      );
+    });
+    it("check isolated pools", async () => {
+      checkIsolatedPoolsComptrollers();
```

I would specify a USDC supplier, to actually execute these checks. Otherwise, they are silently skipped.